src/main/kotlin/org/rust/lang/utils/RsDiagnostic.kt

```diff
@@ -1185,6 +1185,26 @@ sealed class RsDiagnostic(
             )
         }
     }
+
+    class MissingFieldsInUnionPattern(pat: RsPat) : RsDiagnostic(pat) {
+        override fun prepare(): PreparedAnnotation {
+            return PreparedAnnotation(
+                ERROR,
+                NONE,
```

Comment:
JFYI: I think it's strange that there is no error code for this case. So I've opened the corresponding [issue](https://github.com/rust-lang/rust/issues/70620)